scripts/install.sh

```diff
@@ -19,8 +19,9 @@ set -ev
 pip install tox
 
 # App Engine tests require the App Engine SDK.
-if [[ "${TOX_ENV}" == "gae" && ! -d ${GAE_PYTHONPATH} ]]; then
-    python scripts/fetch_gae_sdk.py `dirname ${GAE_PYTHONPATH}`
+if [[ "${TOX_ENV}" == "gae" || "${TOX_ENV}" == "cover" ]]; then
+    pip install git+https://github.com/GoogleCloudPlatform/python-repo-tools.git
```

Comment:
RE: root, it matters. If you can't `pip install` (due to not being root) this will fail